packages/binaryen/binaryen.0.11.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings for Binaryen"
+maintainer: "oscar@grain-lang.org"
+authors: "Oscar Spencer"
+license: " Apache-2.0"
+homepage: "https://github.com/grain-lang/binaryen.ml"
+bug-reports: "https://github.com/grain-lang/binaryen.ml/issues"
+depends: [
+  "ocaml" {>= "4.12" & < "4.13"}
```

Comment:
I think there was a reason originally when I put this together, but maybe it was solved in the meantime. I can't remember what it was because this release has taken months to come together.

Reply:
> this release has taken months to come together.

what do you mean?

Reply:
Nothing. It's just been months to get this all merged and released.